app/metrics.yaml

```diff
@@ -130,3 +130,66 @@ settings_screen:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-08-17"
+
+shortcuts:
+  shortcuts_on_home_number:
```

Comment:
Since this is being sent on the metrics ping, which is transmitted at most, once a day, this will only end up recording the _last_ value that was set by the app. If you need to see how many shortcuts there are per "session", then you would want to create a custom ping and include this metric in it. Then you can send the custom ping after you have recorded this (and any other "session" type metrics).

Reply:
I think we only need to have an average on how many shortcuts users have on the home screen, so IMO a once per day would be enough as it would. I also think this would also add more consistency to the statistics, excluding cases where people add or remove shortcuts in each session.